src/Promitor.Core.Scraping/Configuration/Serialization/v1/Model/LogAnalyticsDefinitionV1.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+
+namespace Promitor.Core.Scraping.Configuration.Serialization.v1.Model
+{
+    public class LogAnalyticsDefinitionV1
```

Comment:
I don't think this is accurate, is it? From the sample I've noticed that only `Query` is used

Reply:
Sorry we don't need this file. Deleted